### DISCUSSION TITLE
help me about ISA categories ?

### DISCUSSION THREAD

**Discussion by link4alink4a** [riscv/riscv-isa-manual#1232] [Q&A]
hi,.everyone, I recently started to learn about ISA manual, so, could u tell me that if  the isa manual volume 1 could be viewed as one single specification?

**Comment by aswaterman** [comment]
See discussion on https://github.com/riscv/riscv-isa-manual/discussions/986#discussioncomment-8495471

**Comment by Zoro210** [comment]
I was going through isa instructions, can someone explain the use of auipc of u type with an example.
